### PR TITLE
[cirque] Basic Cluster Attributes test

### PR DESCRIPTION
**Pull request by doublemis1**
### Change overview
 * Refactor python chip controller test_scripts
 * Extending the MobileDeviceTest to Basic Cluster Attributes verification

### Testing
Verified on Linux machine (Ubuntu 20.04)

scripts/tests/cirque_tests.sh run_all_tests
Logs will be stored at /tmp/tmp.xdbjrcbhN2
[ RUN] EchoTest
[PASS] EchoTest
[ RUN] MobileDeviceTest
[PASS] MobileDeviceTest
[ RUN] InteractionModelTest
[PASS] InteractionModelTest
[PASS] Test finished, test log can be found at /tmp/tmp.xdbjrcbhN2

### CODE REVIEW

**Review by msandstedt — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

File: scripts/tests/cirque_tests.sh [hunk 1]
```diff
@@ -37,6 +37,7 @@ CIRQUE_TESTS=(
     "EchoTest"
     "MobileDeviceTest"
     "InteractionModelTest"
+    "BasicClusterAttributes"
```
Comment (yunhanw-google) [inline scripts/tests/cirque_tests.sh:40]: Could you add some description on this test, per offline message, I think this is used to validate cluster between border router and end device?

File: src/test_driver/linux-cirque/test-basic-cluster-attrs.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+sh = logging.StreamHandler()
+sh.setFormatter(
+    logging.Formatter(
+        '%(asctime)s [%(name)s] %(levelname)s %(message)s'))
+logger.addHandler(sh)
+
+DEVICE_CONFIG = {
+    'device0': {
+        'type': 'MobileDevice',
```
Comment (yunhanw-google) [inline src/test_driver/linux-cirque/test-basic-cluster-attrs.py:36]: Should this be Border router topology? Later you also need to validate SRP in this node?

Comment (doublemis1) [reply src/test_driver/linux-cirque/test-basic-cluster-attrs.py:36]: This test case does not need the border router.
The SRP is not part of this test.

**Review by yunhanw-google — APPROVED** [review]

File: scripts/tests/cirque_tests.sh (hunk 1 above)
Comment (yunhanw-google) [reply scripts/tests/cirque_tests.sh]: never mind, per offline chat, this would not cover border router test. thanks

**Review by andy31415 — APPROVED** [review]
Could you confirm in the PR summary under a testing section that you ran this and it passed (or any other things you did to validate the change)?

I belive cirque is still disabled in our current CI

**Comment by doublemis1** [comment]
> Could you confirm in the PR summary under a testing section that you ran this and it passed (or any other things you did to validate the change)?
> 
> I belive cirque is still disabled in our current CI

Done

**Pull request merged**